packages/suite/src/actions/settings/deviceSettingsActions.ts

```diff
@@ -62,6 +62,39 @@ export const changePin =
         }
     };
 
+export const changeWipeCode =
+    (params: Parameters<typeof TrezorConnect.changeWipeCode>[0] = {}, skipSuccessToast?: boolean) =>
```

Comment:
@mroz22 
I'm trying to build on changePin "template".. however don't know why `TrezorConnect.changeWipeCode` is not reachable?